docs/Converters/UKBtoBIDS.md

```diff
@@ -1,70 +1,83 @@
-- T1W (nifti)
-- T2 Flair (nifti)
-- DWI (nifti)
-- SWI (nifti)
-- tfMRI (dicom)
-- rsfMRI (dicom)
+### Supported modalities
 
-!!! Chosen files    
-    Whenever is possible, the converter uses the rawest files found. This decision allows the user to choose the processing needed. When available the converter get the associated json.
-| Modality    | Chosen image(s) | Justification |
-| :----------:|:---------------:|:-------:|
-| T1W                     | T1.nii.gz       | Defaced and cropped so there is no neck. The rawest image would be the simply defaced one, but brain studies usually are not interested in the region of the neck. No other corrections are included. |
-| T2 Flair    | T2_FLAIR.nii.gz | Defaced and cropped so there is no neck. The reasons for choosing this image are the same as for T1. |
-| DWI         | AP/PAnii.gz     | It is the rawest images we can get. bvals and bvec are also available. |
-| rsfMRI      | rsfMRI.dcm | Since the nii.gz version doesn't always include a json and in consideration for it's usage, we convert the dicom. |
-| tfMRI       | tfMRI.dcm   | Same as rsfMRI.|
-| SWI         | SWI.nii.gz      | Combined coil version. We would get a rawer version, but `dcm2niix` has trouble handling the slices direction, so it is simpler to go with this version. In addition, SWI modality is not fully integrated to BIDS specification and some changes may be coming. A simple version is available in the current version of this converter. | 
-    
+<div class="annotate" markdown>
+Please note that this converter processes the following modalities (1) :
+</div>
 
-## Using the converter
+1. Whenever possible, the converter uses the rawest files available. This decision allows the user to choose the processing they need. If possible the converter also gets the associated json.
+
+| Modality    | Chosen image | Format |                                                                                                                                                              Justification                                                                                                                                                               |
+| :----------:|:---------------:|:-------:|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|
+| T1W         | T1.nii.gz       | nifti |                                                                  Defaced and cropped so there is no neck. The rawest image would be the simply defaced one, but brain studies usually are not interested in the region of the neck. No other corrections are included.                                                                   |
```

Comment:
Would be awesome to add the other abbreviations to `abbreviations.md` 🤩

Reply:
Will do !!